src/rpc/blockchain.cpp

```diff
@@ -84,12 +84,14 @@ double GetNetworkDifficulty(const CBlockIndex* blockindex)
 }
 
 static UniValue ValuePoolDesc(
-    const std::string &name,
+    const std::optional<std::string> name,
     const std::optional<CAmount> chainValue,
     const std::optional<CAmount> valueDelta)
 {
     UniValue rv(UniValue::VOBJ);
-    rv.pushKV("id", name);
+    if (name.has_value()) {
+        rv.pushKV("id", name.value());
+    }
     rv.pushKV("monitored", (bool)chainValue);
     if (chainValue) {
```

Comment:
```suggestion
    if (chainValue.has_value()) {
```
and similarly `if (valueDelta.has_value())` below.

Reply:
Since these lines are unchanged, I'm going to leave them as they are.